### PR TITLE
added:login page & Toast

### PR DESCRIPTION
**Pull request by devsurajrai**
I Have added the login page and the toast for my video library. Please Review

Live: https://deploy-preview-8--quiet-meerkat-2d9ef3.netlify.app/

Screenshot 2022-05-26 at 6 17 22 PM

Screenshot 2022-05-26 at 6 20 29 PM

### CODE REVIEW

File: src/Components/LoginMain.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Input, Button } from "./components.js";
+import { useDispatch } from "react-redux";
+import { selectToastData, setToastData } from "../redux/slice/toastSlice.js";
+import { useSelector } from "react-redux";
+import { useState, useEffect } from "react";
+const LoginMain = () => {
+  const dispatch = useDispatch();
```
Comment (Ramandeep9898) [inline src/Components/LoginMain.jsx:7]: can name dispatch better

**Review by palash23jain — APPROVED** [review]
changes look good to me, good clean coding practice

**Review by Ramandeep9898 — COMMENTED** [review]
clean code and ui ... LGTM!!

**Review by shraddha15-creator — APPROVED** [review]
code looks clean
good folder structure

File: src/Components/LoginMain.jsx (hunk 1 above)
Comment (devsurajrai) [reply src/Components/LoginMain.jsx:7]: Thanks. But, useDispatch always returns the function which is then used to dispatch action creators so there is no need for more declarative naming ig.

**Pull request merged**